fuse_models/src/acceleration_2d.cpp

```diff
@@ -71,6 +72,11 @@ void Acceleration2D::onInit()
     ROS_WARN_STREAM("No dimensions were specified. Data from topic " << ros::names::resolve(params_.topic) <<
                     " will be ignored.");
   }
+
+  if (!params_.target_frame.empty())
+  {
+    tf_listener_ = std::make_unique<tf2_ros::TransformListener>(tf_buffer_);
```

Comment:
I'm assuming one of the reasons we're seeing performance issues is that we only care about static transforms in some cases, and the listener is having to deserialize all transforms. Would it be worth adding a param to the sensor models that indicate we only want static transforms? Rolling [has that option now].(https://github.com/ros2/geometry2/blob/7767a7072e7fb9acb7d18127984fccee048de060/tf2_ros/src/transform_listener.cpp#L43) It forces the listener to only subscribe to `/tf_static`.